### ISSUE TITLE
[Analyzer] `InitializeParams` type missing from `index.d.ts`

### ISSUE THREAD

**Issue by xenobytezero** [open-wc/custom-elements-manifest#237]
**Checklist**
- [ x] Did you run the analyzer with the `--dev` flag to get more information? - N/A
- [ x] Did you create a **minimal** reproduction in the [playground](https://custom-elements-manifest.netlify.app/)? - N/A

I'm importing the `@custom-elements-manifest/analyzer` types to help writing a plugin, and `tsc` is complaining the following 

```
../../../node_modules/.pnpm/@custom-elements-manifest+analyzer@0.9.2/node_modules/@custom-elements-manifest/analyzer/index.d.ts:95:23 - error TS2304: Cannot find name 'InitializeParams'.

95   initialize?(params: InitializeParams): void;
                         ~~~~~~~~~~~~~~~~


Found 1 error in ../../../node_modules/.pnpm/@custom-elements-manifest+analyzer@0.9.2/node_modules/@custom-elements-manifest/analyzer/index.d.ts:95
```

Sure enough, this type is missing, and is flagged as an error when opening the file up in VSCode. 

I can work around the issue using `skipLibCheck` but I am assuming this omission is a mistake, and thought I should flag it.

**Comment by thepassle** [comment]
Ah yes, we recently added this hook but I guess forgot to update the .d.ts file. Would you mind creating a PR? :)

**Comment by xenobytezero** [comment]
Will do.

**Comment by xenobytezero** [comment]
Done!

**Comment by thepassle** [comment]
Released in 0.9.3
